packages/appium/docs/en/quickstart/test-py.md

```diff
@@ -2,4 +2,51 @@
 title: Write a Test (Python)
 ---
 
-TODO
+[Appium Python Client](https://github.com/appium/python-client) is
+an official Appium client in Python, which is available via pypi as [Appium-Python-Client](https://pypi.org/project/Appium-Python-Client/) package name.
+It inherits [Selenium Python Binding](https://pypi.org/project/selenium/),
+so installing the Appium Python Clint includes the selenium binding.
+
+```bash
+pip install Appium-Python-Client
+```
+
+This example follows `unittest` module.
```

Comment:
`This example uses Python's built-in unittest module, though you can use any Python test framework you want.`